shared/modules/logic/constants.ts

```diff
@@ -0,0 +1,39 @@
+import { BasicField } from '../../types/field'
+import { LogicCondition, LogicConditionState } from '../../types/form'
+
+const LOGIC_CONDITIONS: LogicCondition[] = [
```

Comment:
it was in `src/shared/util` (the fake shared directory), meaning react doesn't have access to it. Instead of making this PR even larger by moving the whole of src/shared/logic into shared and deciding what is needed is what is not, I just imported the relevant stuff first. We can then decide what is actually shared and what is actually server-side code in future PRs